docs/source/tutorials/e2e_flow.rst

```diff
@@ -250,76 +267,76 @@ For this, we'll use the
+        path: ${output_dir}/original/tokenizer.model
+        prompt_template: null
 
-        # checkpoint files for the fine-tuned model. This should
-        # match what's shown in the logs above
+    model:
+        # Notice that we don't pass the lora model. We are using the merged weights,
+        _component_: torchtune.models.llama3_2.llama3_2_3b
+
+    checkpointer:
+        _component_: torchtune.training.FullModelHFCheckpointer
+        checkpoint_dir: ${output_dir}
         checkpoint_files: [
-            hf_model_0001_0.pt,
-            hf_model_0002_0.pt,
+            ft-model-00001-of-00002.safetensors,
+            ft-model-00002-of-00002.safetensors,
         ]
+        output_dir: ${output_dir}
+        model_type: LLAMA3_2
 
-        output_dir: <checkpoint_dir>
-        model_type: LLAMA2
+    ### OTHER PARAMETERS -- NOT RELATED TO THIS CHECKPOINT
 
-    # Make sure to update the tokenizer path to the right
-    # checkpoint directory as well
-    tokenizer:
-        _component_: torchtune.models.llama2.llama2_tokenizer
-        path: <checkpoint_dir>/tokenizer.model
+    device: cuda
+    dtype: bf16
+
+    seed: 1234
+
+    # Generation arguments; defaults taken from gpt-fast
+    prompt:
+    system: null
+    user: "Tell me a joke. "
+    max_new_tokens: 300
+    temperature: 0.6 # 0.8 and 0.6 are popular values to try
+    top_k: 300
 
+    enable_kv_cache: True
+
+    quantizer: null
 
 Once the config is updated, let's kick off generation! We'll use the
 default settings for sampling with ``top_k=300`` and a
 ``temperature=0.8``. These parameters control how the probabilities for
-sampling are computed. These are standard settings for Llama2 7B and
-we recommend inspecting the model with these before playing around with
+sampling are computed. We recommend inspecting the model with these before playing around with
 these parameters.
 
-We'll use a different prompt from the one in the config
-
-.. code-block:: bash
-
-    tune run generate --config ./custom_generation_config.yaml \
-    prompt="What are some interesting sites to visit in the Bay Area?"
-
-
-Once generation is complete, you'll see the following in the logs.
-
-
-.. code-block:: bash
-
-    [generate.py:92] Exploratorium in San Francisco has made the cover of Time Magazine,
-                     and its awesome. And the bridge is pretty cool...
-
-    [generate.py:96] Time for inference: 11.61 sec total, 25.83 tokens/sec
-    [generate.py:99] Memory used: 15.72 GB
+.. code-block:: text
 
+    $ tune run generate --config ./custom_generation_config.yaml prompt="tell me a joke. "
+    Tell me a joke. Here's a joke for you:
 
-Indeed, the bridge is pretty cool! Seems like our LLM knows a little something about the
-Bay Area!
+    What do you call a fake noodle?
 
-|
+    An impasta!
 
-Speeding up Generation using Quantization
------------------------------------------
+Introduce some quantization
```

Comment:
our quantize recipe only outputs 1 file that is .pt. I am tempted to say that we should remove this from here until we have better support for quantization, e.g. output safetensors and multiple checkpoints. Any thoughts?